TimeFormat.java

```diff
@@ -1,17 +1,26 @@
 // Represents the hh:mm time format using an AM/PM format. 
 public class TimeFormat {
 	public static void main(String[] args) {
-		// In Java, the command-line arguments args[0], args[1], ... are represented
-		// each by a string. In this program, the single "hh:mm" input is represented
-		// by the single command-line string argument args[0]. 
-		//   
-		// The following statement handles the hours part of the input.
-		// It concatenates the empty string "" with the leftmost hour-digit. 
-		// It then concatenates the resulting string with the rightmost hour-digit,
-		// and then uses parseInt to cast the resulting string as an int.
+		
 		int hours = Integer.parseInt("" + args[0].charAt(0) + args[0].charAt(1));
-		// Does the same with the minutes part of the input.
 		int minutes = Integer.parseInt("" + args[0].charAt(3) + args[0].charAt(4));
-        // Replace this comment with the rest of your code
+        if (minutes>=10)

```

Comment:
Spacing (in else, in the first if and along the code) + bad indentations
Aside from that: 
you have here repeated code. 
Example for improvement:

```
String minutesStr = "";
String hoursStr = "";
String amPmStr = "";
if (minutes >= 10) {
   minutesStr += minutes;
} else {
   minutesStr += "0" + minutes;
}

if (hours <= 12) {
    hoursStr += hours; 
} else {
    hoursStr += (hours - 12); 
}

if (hours < 12){
   amPmStr += "AM";   
} else {
   amPmStr += "PM";
}

System.out.println(hoursStr + ":" + minutesStr + " " + amPmStr);
```